### DISCUSSION TITLE
Where do I find ClamAV logs?

### DISCUSSION THREAD

**Discussion by ShapeShifter499** [nextcloud/all-in-one#4162] [Questions] [answered]
I'm a bit confused. Where do I look for the ClamAV log? Occasionally, I'll get a nextcloud notification, but if those get dismissed in the WebUI or from the desktop client, I'm not sure where to find those again.  Do I really have to dig through each nextcloud log to see what files were flagged as possible viruses found?

**Comment by szaimen** [comment, marked as answer]
> Do I really have to dig through each nextcloud log to see what files were flagged as possible viruses found?

Yes
